### DISCUSSION TITLE
0.11.0-stable has been released!

### DISCUSSION THREAD

**Discussion by mihe** [godot-jolt/godot-jolt#701] [Announcements]
This release only supports **Godot 4.2** (including 4.2.x).

## Changelog

These are the notable changes that have been made since [0.10.0-stable](https://github.com/godot-jolt/godot-jolt/releases/tag/v0.10.0-stable) was released. You can also find a list of all the commits [here](https://github.com/godot-jolt/godot-jolt/compare/v0.10.0-stable...v0.11.0-stable).

Any breaking changes are denoted with ⚠️.

### Changed

- ⚠️ Changed `HeightMapShape3D` to always use back-face collision, to match Godot Physics.

### Fixed

- Fixed issue with project randomly freezing up when having many active physics spaces.
- Fixed issue with static and kinematic bodies not correctly incorporating surface velocities, also known as "constant velocities", as part of their reported velocities. This also makes it so `move_and_slide` will respect such velocities.
- Fixed issue with global transform not being preserved when reparenting a `RigidBody3D`.
- Fixed issue where the callback passed to `body_set_force_integration_callback` could be called even when the body is sleeping.

<hr /><em>This discussion was created from the release <a href='https://github.com/godot-jolt/godot-jolt/releases/tag/v0.11.0-stable'>0.11.0-stable</a>.</em>

**Comment by Gamepro5** [comment]
That was fast, thank you!

**Comment by jitspoe** [comment]
Thanks for your dedication to this project!
